web-client/src/views/TrialSessionWorkingCopy/SessionNotes.tsx

```diff
@@ -19,54 +19,52 @@ export const SessionNotes = connect(
   }) {
```

Comment:
Nitpick, but probably something UX will notice: there is still a small window in which the buttons wrap above the header.

<img width="700" alt="Screenshot 2024-11-20 at 9 44 06 AM" src="https://github.com/user-attachments/assets/8ecb41a0-a61a-44c5-a862-d49bb0950b9f">

Also, I know that the reason for the setup being the way it is here comes about because the CSS in our app is a convoluted mess, but I wonder if there is a way to put the header and buttons inline at the top of the component? Semantically, I expect to see the header row at the top and the note text beneath.

Reply:
Thanks for catching that! could you clarify what you mean by "a small window" are you referring to a specific screen size or resolution?